src/unbranded/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="scheme">https</string>
+	<string name="fixed_app_url"></string>
+</resources>
```

Comment:
This file is probably not needed if `scheme` is in the `src/main/res/values/strings.xml` file.

Reply:
Removed.